src/weather.js

```diff
@@ -0,0 +1,20 @@
+class Weather {
+  constructor(city) {
+    this.apiKey = 'f374063fac3fb5e66f52e39195a343bd';
+    this.city = city;
+  }
+
+  async getWeather() {
+    const response = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${this.city}&appid=${this.apiKey}`,
+    );
+    const responseData = await response.json();
+
+    return responseData;
+  }
+
+  changeLocation(city) {
+    this.city = city;
```

Comment:
According to the previous review, @Torres-ssf mentioned calling different `API`

kindly check the screenshot from the pathwright below;
![image](https://user-images.githubusercontent.com/31897434/84189734-a6933f80-aa8d-11ea-8d60-7dc908bbf80a.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/